src/python/ensembl/io/genomio/gff3/process_gff3.py

```diff
@@ -25,14 +25,12 @@
 import json
 import argschema
 
-
 from BCBio import GFF
 from Bio.SeqRecord import SeqRecord
 from Bio.SeqFeature import SeqFeature
 
 from ensembl.io.genomio.gff3.functional_annotation import FunctionalAnnotations
 
-
```

Comment:
It's been some time, but I believe this empty line is suggested by PEP8 (so if you run `black` it may add it back).